### PR TITLE
Added required permissions for the performer user #1581

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving Issue #1581 with an important note.

Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: migration/how-to-use-the-sharepoint-migration-tool.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ Review the system requirements, settings, and permissions behavior before beginn
 - [SPMT Settings](spmt-settings.md)</br>
 - [Understanding permissions when using the SharePoint Migration Tool](understanding-permissions-when-migrating.md)</br>
 
+ > [!IMPORTANT]
+ > - The required permissions at tenant level for the performer user is "SharePoint Admin".
```
Comment (JohanFreelancer9) [inline migration/how-to-use-the-sharepoint-migration-tool.md]:
```suggestion
 > - The required user permissions at tenant level is "SharePoint Admin".
```

File: migration/how-to-use-the-sharepoint-migration-tool.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ Review the system requirements, settings, and permissions behavior before beginn
...
+ > - The required permissions at site collection level for the performer user is "Admin".
```
Comment (JohanFreelancer9) [inline migration/how-to-use-the-sharepoint-migration-tool.md]:
```suggestion
 > - The required user permissions at site collection level is "Admin".
```

**Comment by j0rt3g4** [comment]
All updated @JohanFreelancer9  Thank you!

File: migration/how-to-use-the-sharepoint-migration-tool.md (hunk 1 above)
Comment (JohanFreelancer9) [inline migration/how-to-use-the-sharepoint-migration-tool.md]:
```suggestion
 > - The required permission at tenant level for the performer user is "SharePoint Admin".
```

File: migration/how-to-use-the-sharepoint-migration-tool.md (hunk 2 above)
Comment (JohanFreelancer9) [inline migration/how-to-use-the-sharepoint-migration-tool.md]:
```suggestion
 > - The required permission at site collection level for the performer user is "Admin".
```

**Comment by j0rt3g4** [comment]
@JohanFreelancer9 It's all good, it's fixed now :)

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**